### PR TITLE
Fix pgsql backend compilation on centos 6

### PR DESCRIPTION
**Pull request by pieterlexis**

### CODE REVIEW

File: modules/gpgsqlbackend/spgsql.cc [hunk 1]
```diff
@@ -96,10 +96,16 @@ class SPgSQLStatement: public SSqlStatement
     // by stored procedures. you can return more than one
     // if you return SETOF refcursor.
     if (PQftype(d_res_set, 0) == 1790) { // REFCURSOR
+#if PG_VERSION_NUM > 90000
+      // PQescapeIdentifier was added to libpq in postggresql 9.0
       char *val = PQgetvalue(d_res_set, d_cur_set++, 0);
       char *portal =  PQescapeIdentifier(d_db(), val, strlen(val));
       string cmd = string("FETCH ALL FROM \"") + string(portal) + string("\"");
       PQfreemem(portal);
```
Comment (pieterlexis) [inline modules/gpgsqlbackend/spgsql.cc]: I _think_ this line should be after `#endif`

**Comment by cmouse** [comment]
Hopefully the portal value won't contain escapable characters...

**Comment by ahupowerdns** [comment]
is this ok for merge?

File: modules/gpgsqlbackend/spgsql.cc (hunk 1 above)
Comment (rubenk) [inline modules/gpgsqlbackend/spgsql.cc]: Pff, what I meant to say was, there comment should say PostgreSQL, not postggresql.

**Pull request merged**